pkg/checker/host_checker.go

```diff
@@ -31,9 +31,16 @@ type HostChecker struct {
 
 func (a HostChecker) Check(cluster *v2.Cluster, phase string) error {
 	var ipList []string
+	var masterNum int
 	for _, hosts := range cluster.Spec.Hosts {
+		if len(hosts.Roles) > 0 && hosts.Roles[0] == v2.MASTER {
+			masterNum++
+		}
 		ipList = append(ipList, hosts.IPS...)
 	}
+	if masterNum & 1 == 0 {
+		return fmt.Errorf("checker: occurs no-odd number of master nodes")
```

Comment:
Maybe we should use the [`confirm function`](https://github.com/labring/sealos/blob/339adfa18ca0af11c49c33c1c5695655e74db2ca/pkg/utils/confirm/confirm.go#L25) to warn the user and let the user make the final decision.
https://github.com/labring/sealos/blob/339adfa18ca0af11c49c33c1c5695655e74db2ca/pkg/utils/confirm/confirm.go#L25

Reply:
fine, I will check it out.